distribution/src/main/resources/Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM alfresco/alfresco-base-java:jre17-rockylinux8-202302221525
+# ACTUAL IMAGE
+# More infos about this image: https://github.com/Alfresco/alfresco-docker-base-java
+# jre17-rockylinux8 (29/07/2024)
+FROM alfresco/alfresco-base-java@sha256:18ac53e9a267dc4389eaff80ad1f5b8deaf53da09c234de5fcc6e05ebcd4a77c
```

Comment:
Wrong image? AFAIR we're bumping everything to RL9 and this is still for RL8, ref: https://github.com/Alfresco/alfresco-enterprise-repo/pull/2127/files